internal/cli/universal_login.go

```diff
@@ -306,183 +293,9 @@ func updateBrandingTemplateCmd(cli *cli) *cobra.Command {
 		Example: `  auth0 universal-login templates update
   auth0 ul templates update`,
 		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Should we print a warning saying that the command is deprecated, and to use the new no-code editor in the Dashboard instead?